pyproject.toml

```diff
@@ -71,6 +72,10 @@ ignore_whitespace = true
 module = ["brownie.*", "twisted.*"]
 ignore_missing_imports = true
 
+[[tool.mypy.overrides]]
+module = ["beamer.tests.*"]
+disallow_untyped_defs = false
+
```

Comment:
Why did you add this? Perhaps a `rebase` artifact?

Reply:
yes, somehow it ended up there.